editor.planx.uk/src/components/Header.tsx

```diff
@@ -460,11 +466,33 @@ const EditorToolbar: React.FC<{
         }}
       >
         <StyledPaper>
-          {/*
-          <MenuItem onClick={() => handleClick("/")}>Service settings</MenuItem>
-          <MenuItem onClick={() => handleClick("/")}>My dashboard</MenuItem>
-           */}
-
+          <MenuItem disabled>
+            <ListItemIcon>
+              <Person fontSize="small" />
+            </ListItemIcon>
+            <ListItemText>
+              {user.email}
+            </ListItemText>
+          </MenuItem>
+          {(user.isPlatformAdmin || user.teams.length > 0) && (
+            <MenuItem disabled>
+              <ListItemIcon>
+                <Edit />
+              </ListItemIcon>
+              <ListItemText>
+                {user.isPlatformAdmin ? `All teams` : user.teams.map((team) => team.team.name).join(", ")}
+              </ListItemText>
+            </MenuItem>
+          )}
+          <MenuItem disabled divider>
+            <ListItemIcon>
+              <Visibility />
+            </ListItemIcon>
+            <ListItemText>
+              All teams
+            </ListItemText>
+          </MenuItem>
```

Comment:
`disabled` menu items definitely isn't a11y compliant, but was a quick way to get consistent styling here & editor doesn't have to comply with any audits yet